src/ui/toolbar/components/ToolbarColorPicker.vue

```diff
@@ -47,8 +50,11 @@
         <div
           v-for="(color, index) in colorPalette"
           :key="index"
+          role="gridcell"
           class="c-palette__item"
           :style="{ background: color.value }"
+          :title="color.value"
+          :aria-label="color.value"
```

Comment:
presents the hex value of the color to the user. Given that this is a very "visual" component, I assume that the screenreader would know what to do with this